buku

```diff
@@ -4014,35 +4014,21 @@ def import_html(html_soup: BeautifulSoup, add_parent_folder_as_tag: bool, newtag
             desc = comment_tag.find(string=True, recursive=False)
 
         if add_parent_folder_as_tag:
-            # add parent folder as tag
```

Comment:
Removed this comment (since it was merely quoting the line immediately before it)